node/setup.go

```diff
@@ -594,6 +594,12 @@ func LoadStateFromDBOrGenesisDocProviderWithConfig(
 		return sm.State{}, nil, ErrGenesisDoc{Err: err}
 	}
 
+	checkSumStr := hex.EncodeToString(csGenDoc.Sha256Checksum)
+	if err := tmhash.ValidateSHA256(checkSumStr); err != nil {
+		const formatStr = "invalid genesis doc SHA256 checksum: %s"
```

Comment:
What's the benefit of this? If the `const` is not reused anywhere?

Reply:
It shows me at a glance that `formatStr` is immutable, plus there is a performance benefit for using `const`, albeit  _minor_ in this case.
Tbh, it's my code-style preference: if a function call using a `string` is long enough to wrap to a new line in my editor, I create a separate `string` like I did here. It's all for readability.